### PR TITLE
Adds grouped entry + RTL section + additional tests

### LINKED ISSUE #4127: Template Review
Please review the following template, and comment on whether the steps of what is being tested or needs to be tested is well described enough.

This issue is to check whether this template provides enough detail for someone else to complete manual testing of the app.

# :penguin: Manual Test Template

## Testing sign in area ##

- [ ] Test standard email and password sign in
- [ ] Test standard email and password sign in with invalid email address
- [ ] Test standard email and password sign in with invalid password
- [ ] Test signup with standard email and password
- [ ] Test terms and conditions link
- [ ] Test privacy policy link
- [ ] Test signin with Google signin
- [ ] Test signup with Google signin
- [ ] Test forgotten password link
- [ ] Test forgotten password link with invalid email
- [ ] Is country selection automatically selecting the right country?
- [ ] Is country selection allowing manual selection
- [ ] Test email field with invalid email
- [ ] Test email field with mixed character sets (including various languages)
- [ ] Test password field for old/changed password

## Testing timer page
- [ ] Start timer
- [ ] Stop timer
- [ ] Enter description
- [ ] Add new project
- [ ] Add new client
- [ ] Add new tag
- [ ] Add long description and confirm character limit error shows
- [ ] Add new project using existing client
- [ ] Add existing project
- [ ] Add existing tag
- [ ] Start new timer, select no project, stop
- [ ] Create new manual time entry
- [ ] Start a new entry, add multiple tags.
- [ ] Start a new entry, discard new entry.
- [ ] Continue an entry
- [ ] Delete an entry

## Testing editing time entry
- [ ] Change description
- [ ] Change project
- [ ] Remove project
- [ ] Change tags
- [ ] Remove tags
- [ ] Change start time with the barrel/manual selection
- [ ] Change start time with the wheel
- [ ] Change end time with barrel/manual selection
- [ ] Change start date
- [ ] Change end date
- [ ] Change duration
- [ ] Change date from main entry page
- [ ] Discard changes to entry
- [ ] Delete entry from edit view
- [ ] Confirm changes
- [ ] Create new project from edit view
- [ ] Create new client from edit view
- [ ] Create new tag from edit view

## Testing report screen
- [ ] Custom range
- [ ] Is correct data displayed
- [ ] Are categories displaying correct timerframe and dates?
- [ ] Is web displaying the same data?
- [ ] Today
- [ ] Yesterday
- [ ] This year
- [ ] This month
- [ ] Last month
- [ ] This week
- [ ] Last week
- [ ] Custom range 1 year
- [ ] Custom range 1 month
- [ ] Custom range 1 week
- [ ] Custom range 1 day
- [ ] Ensure correct year is selected on all options
- [ ] Change workspaces

## Testing settings page
- [ ] Sign out normal email
- [ ] Sign out google sign in
- [ ] Change workspace
- [ ] Date format MM/DD/YYYY
- [ ] Date format DD-MM-YYYY
- [ ] Date format MM-DD-YYYY
- [ ] Date format YYYY-MM-DD
- [ ] Date format DD/MM/YYYY
- [ ] Date format DD.MM.YYYY
- [ ] 24 hour time format
- [ ] 12 hour format
- [ ] Duration format Classic
- [ ] Duration format Improved
- [ ] Duration format Decimal
- [ ] First day of the week Monday
- [ ] First day of the week Tuesday
- [ ] First day of the week Wednesday
- [ ] First day of the week Thursday
- [ ] First day of the week Friday
- [ ] First day of the week Saturday
- [ ] First day of the week Sunday
- [ ] Manual mode
- [ ] Submit feedback
- [ ] About
- [ ] Help
- [ ] Is "Synced" displaying correctly?

## Testing Calendar Integration
- [ ] Are calendar entries showing correctly?
- [ ] Create an entry from a calendar entry
- [ ] Adjust the start time
- [ ] Adjust the end time

## Testing Siri Integration
- [ ] Start a time entry
- [ ] Stop a time entry

## Multiple Workspaces
- [ ] Switch workspaces
- [ ] Go offline then switch workspaces (Airplane mode)
- [ ] Switch between workspaces while timer running
- [ ] Is data displaying correctly in web?
- [ ] Force sync then switch
- [ ] Leave a workspace
- [ ] Be removed from a workspace while offline
- [ ] Be removed from a workspace with an active entry running

## Testing Interactions/Sync between this app and others
- [ ] Edit client on web whilst mobile online
- [ ] Delete client on web whilst mobile online and tracking a task
- [ ] Edit client on web whilst mobile online and tracking a task
- [ ] Tags
- [ ] Archiving
- [ ] Create new project on web whilst mobile offline
- [ ] Tasks
- [ ] Try creating new entries on web whilst mobile offline
- [ ] Edit client on web whilst mobile offline
- [ ] Try stopping timer on web while mobile offline
- [ ] Delete project on web whilst mobile offline
- [ ] Make changes to projects on web

## Testing in different timezones

- [ ] Check

## Testing onboarding

- [ ] Check

## Test that the UI is appearing as intended and report any issues

- [ ] Check

### PR DESCRIPTION
**Pull request by ApocalypticFish**
This adds a section for testing grouped entries to the Manual testing template, it also adds a section for RTL languages and some tests to both.
Added some additional tests for timezones, calendar integration and the Siri integration.

## What's this?
It updates the Manual testing template

### Relationships
Closes #4127 
Similar issues will be periodically created as the app changes

## Why do we want this?
Because it brings the test up to date with recent changes to the app

## How is it done?
It adds several lines of tests to the Manual Testing template

### Why not another way?
Suggestions welcome

### Side effects
None really, slightly longer manual testing but this is necessary to ensure all continues working as the app continues to develop in complexity

## Review hints
Check the doc and if you have any suggestions for things that could also be checked please suggest them.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/d-manual-test-template.md [hunk 1]
```diff
@@ -62,6 +68,20 @@ about: A manual testing checklist that can be assigned to a release to track the
...
 
+## Grouped Time Entries
+- [ ] Make sure the `Group time entries` setting is propagated to the server
+- [ ] Check whether the main log respects the grouping setting
+- [ ] Check whether the group count is correct in both expanded and collapsed groups
+- [ ] Collapse and expand various groups (ones with and without descriptions, tags, or projects, etc.)
+- [ ] Tap to edit a group header
+- [ ] Tap to edit a TE in an expanded group
+- [ ] Swipe to delete a group
+- [ ] Swipe to delete a TE from an expanded group
```
Comment (arturs105) [inline .github/ISSUE_TEMPLATE/d-manual-test-template.md]: Testing the undo deletion feature for groups and individual entries could be useful as well.

File: .github/ISSUE_TEMPLATE/d-manual-test-template.md [hunk 2]
```diff
@@ -107,15 +127,18 @@ about: A manual testing checklist that can be assigned to a release to track the
...
 ## Testing Calendar Integration
 - [ ] Are calendar entries showing correctly?
 - [ ] Create an entry from a calendar entry
-- [ ] Adjust the start time
-- [ ] Adjust the end time
+- [ ] Edit the entry by adjusting the start time of the entry (is this reflected on the calendar page?)
+- [ ] Edit the entry by adjusting the end time (is this reflected on the calendar page?)
+- [ ] Adjust the time of the calendar entry from your calendar to earlier on the current day (is this reflected?)
+- [ ] Delete the calendar entry from your calendar (is this reflected?)
+- [ ] Move the calendar entry from your calendar to another day (is this reflected?)
```
Comment (arturs105) [inline .github/ISSUE_TEMPLATE/d-manual-test-template.md]: Can this even be done, since we only show the current day in calendar?

Comment (kornelijepetak) [reply .github/ISSUE_TEMPLATE/d-manual-test-template.md]: What is the app supposed to do in this case? Just stop showing the calendar entry. It should not touch any TEs, right?

File: .github/ISSUE_TEMPLATE/d-manual-test-template.md (hunk 1 above)
Comment (kornelijepetak) [reply .github/ISSUE_TEMPLATE/d-manual-test-template.md]: Yes, undo feature should be tested as well.

**Comment by kornelijepetak** [comment]
My review can be dismissed during my absence if another dev confirms that my questions were addressed.

File: .github/ISSUE_TEMPLATE/d-manual-test-template.md (hunk 2 above)
Comment (ApocalypticFish) [reply .github/ISSUE_TEMPLATE/d-manual-test-template.md]: Yeah, this is just to confirm that it stops showing the calendar entry

**Pull request merged**